tutorial01_load_vulkan/app/src/main/jni/main.cpp

```diff
@@ -76,15 +76,14 @@ bool initialize(android_app* app) {
     return false;
   }
 
-  VkApplicationInfo appInfo = {
-      .sType = VK_STRUCTURE_TYPE_APPLICATION_INFO,
-      .pNext = nullptr,
-      .apiVersion = VK_MAKE_VERSION(1, 0, 0),
-      .applicationVersion = VK_MAKE_VERSION(1, 0, 0),
-      .engineVersion = VK_MAKE_VERSION(1, 0, 0),
-      .pApplicationName = "tutorial01_load_vulkan",
-      .pEngineName = "tutorial",
-  };
+  VkApplicationInfo appInfo;
+  memset(&appInfo, 0, sizeof(appInfo));
+  appInfo.sType = VK_STRUCTURE_TYPE_APPLICATION_INFO;
```

Comment:
Can't we do:
VkApplicationInfo appInfo = {
       VK_STRUCTURE_TYPE_APPLICATION_INFO,
       nullptr,
[...]

instead ?
This syntax is super verbose...